classes/controllers/FrmFieldsController.php

```
@@ -377,6 +377,67 @@ private static function maybe_define_field_selection_data() {
+		return apply_filters( 'frm_should_hide_field_choice', false, $choice_limit_is_reached, $form_id, $opt_key );
+	}
+
+	/**
+	 * @since x.x
+	 *
+	 * @param array $field
+	 * @return array
+	 */
+	public static function get_choices_limit_reached_statuses( $field ) {
+		$choices_limit_reached_statuses = array();
+		foreach ( $field['options'] as $opt_key => $opt ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$opt'. (undefined)

(UnusedLocalVariable)